bigbluebutton-html5/imports/ui/components/video-dock/video-menu/container.jsx

```diff
@@ -25,13 +45,33 @@ export default withTracker((params) => {
   const user = Users.findOne({ userId: Auth.userID });
   const userLocked = mapUser(user).isLocked;
 
+  const isConecting = (!isSharingVideo && isConnected);
+  const isLocked = (LockCam && userLocked) || webcamOnlyModerator;
+  const isDisabled = isLocked
+      || isWaitingResponse
+      || isConecting
+      || enableShare;
+
+  const baseName = Meteor.settings.public.app.basename;
```

Comment:
Could you move this to the service, please?
As well as `Meetings.findOne`, `Users.findOne` and other collection interactions from above.
We shouldn't interact with the collections right here.